mathml-block.php

```diff
@@ -14,12 +14,58 @@
  *
  * @package mathml-block
  */
+
 namespace MathMLBlock;
 
- /**
-  * Enqueue the admin JavaScript assets.
-  */
+use WP_Block_Type_Registry;
+
+const BLOCK_NAME = 'mathml/mathmlblock';
+
+const MATHJAX_SCRIPT_HANDLE = 'mathjax';
+
+const MATHJAX_SCRIPT_URL = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js';
+
+/**
+ * Determine whether the response will be an AMP page.
+ *
+ * @return bool
+ */
+function is_amp() {
+	return (
+		( function_exists( 'amp_is_request' ) && \amp_is_request() )
```

Comment:
@westonruter is this the newer 2.0+ preferred way to detect is_amp?

Reply:
In reality it is identical. We just renamed `is_amp_endpoint()` to `amp_is_request()` and then added a soft-deprecated `is_amp_endpoint()` which calls `amp_is_request()`. This was done just to start using prefixed functions everywhere. We're not going to hard-deprecate it.

Reply:
👍 ah, ok so purely a naming convention based change. thanks for clarifying.